src/commands/create.js

```diff
@@ -13,65 +12,23 @@ const {
     setLocalConfig,
     setLocalEnv,
     getNpmCmd,
-    validateActorName,
     getJsonFileContent,
     detectPythonVersion,
     isPythonVersionSupported,
     getPythonCommand,
 } = require('../lib/utils');
 const { EMPTY_LOCAL_CONFIG, LOCAL_CONFIG_PATH, PYTHON_VENV_PATH } = require('../lib/consts');
+const { ensureValidActorName, getTemplateDefinition } = require('../lib/create-utils');
 
 class CreateCommand extends ApifyCommand {
     async run() {
         const { flags, args } = this.parse(CreateCommand);
         let { actorName } = args;
-        let { templateArchiveUrl, template: templateName } = flags;
```

Comment:
Please keep the `templateArchiveUrl` argument, it's really useful when developing and testing new templates from a branch.

Reply:
Ah, it was neither documented nor in help so I thought it's redundant. Will put it back.

Reply:
Yeah, it's hidden by design, so that we don't have to support it and keep it for backwards compatibility